m365-developer-proxy-abstractions/PluginEvents.cs

```diff
@@ -187,15 +187,21 @@ public class PluginEvents : IPluginEvents {
     }
 
     public async Task RaiseProxyBeforeRequest(ProxyRequestArgs args) {
-        await BeforeRequest?.InvokeAsync(this, args, null);
+        if (BeforeRequest is not null) {
```

Comment:
Thanks for cleaning these up! It's much appreciated

Reply:
I had a discussion on this topic on X the other day, and we actually prefer the old way, but that causes a warning.